float64 pos_x
float64 pos_y
float64 pos_z
float64 pose_x
float64 pose_y
float64 pose_z
float64 pose_w